tequilapi/endpoints/proposals.go

```diff
@@ -214,6 +224,7 @@ func (pe *proposalsEndpoint) List(resp http.ResponseWriter, req *http.Request, p
 		LowerTimePriceBound: lowerTimePriceBound,
 		UpperTimePriceBound: upperTimePriceBound,
 		ExcludeUnsupported:  true,
+		IncludeFailed:       req.URL.Query().Get("include_failed") == "true",
```

Comment:
`monitoring_failed=true` or `metrics.monitoring_failed=true` would be better name here as:
- as you would not leak API naming to Tequilapi contract
- it's Tequilapi endpoint , so alligned to Tequilapi proposal DTO

Reply:
Renamed.